intel-retail/automated-self-checkout
# feat: object detection script support configurable env file for model related parameters

## PR Checklist



## What are you changing?
modelname in run_object_detection.sh was hardcoded, this PR made the model name configurable to pass in as environment variable to the script.
## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./docker-run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_3 --workload opencv-ovms

above command should run the pipeline successfully without error.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
